File: level 101/classwork/cw2.js
// 3)მომხმარებელს შეეკითხეთ რიცხვი,შემდეგ 1 დან მომხმ შემოტანილ რიცხვამდე დაბეჭდეთ ყველა რიცხვი 

// 4))მომხმარებელს შეეკითხეთ რიცხვი,შემდეგ 1 დან მომხმ შემოტანილ რიცხვამდე დაბეჭდეთ ლუწი რიცხვები

// 5)მომხმარებელს შეეკითხეთ რიცხვი,შემდეგ 1 დან მომხმ შემოტანილ რიცხვამდე დაბეჭდეთ კენტი რიცხვები 
 

// 7)100 დან ქვემოთ დაითვალეთ რიცხვები while loop ის დახმარებით

let userNum=Number(prompt('enter number'))
let  x = 0
while(x<=userNum){
    console.log(x)
    x++
}


let evenNum=Number(prompt('enter numbeer'))
let y=0
while(y<=evenNum){
    if (y %2=== 0){
        console.log(y)
        y++

    }
}


let oddNum=Number(prompt('enter numbeer'))
let  a=0    
while(a<=oddNum){
    if (a%2!=0){
        console.log(a)
        a++

    }
}

File: level 022/classwork/cw2.py
# 2) მომხმარებელს შემოატანინეთ რიცხვი შემდეგ კი ერთიდან ამ რიცხვამდე დაბეჭდეთ მხოლოდ ლუწი რიცხვების ჯამი, გამოიყენეთ for loop

user_num=int(input('enter your number'))

sum=0

for i in range(user_num):
    if i%2==0:
        sum=sum+i
print(sum)